integration-test/src/test/java/com/datastrato/gravitino/integration/test/trino/TrinoConnectorIT.java

```diff
@@ -388,6 +397,14 @@ void testHiveTableCreatedByTrino() {
     Assertions.assertEquals(
         "hdfs://localhost:9000/user/hive/warehouse/hive_schema.db/hive_table",
         table.properties().get("location"));
+    Assertions.assertEquals("MANAGED_TABLE", table.properties().get("table-type"));
```

Comment:
why there are some properties code?

Reply:
I'm having trouble understanding your ideas. Can you provide more detail?